core/storage/voiceover/gae_models.py

```diff
@@ -225,19 +219,8 @@ class VoiceArtistMetadataModel(base_models.BaseModel):
     Instances of this class are keyed by the user ID.
     """
 
-    # Dictionary mapping language codes to nested dictionaries. Each
```

Comment:
Schema updated to store only language code and accent code for a given voice artist. Other information i.e., exploration ID, content ID, and language code are stored in a separate model.